src/zui/ZUITooltip/index.tsx

```diff
@@ -1,30 +1,35 @@
-import { Tooltip, TooltipProps } from '@mui/material';
+import { Tooltip } from '@mui/material';
 import { FC, ReactElement } from 'react';
 
-export interface ZUITooltipProps {
+type ZUITooltipProps = {
+  /** If the tooltip should have a small arrow or not. Defaults to 'true' */
```

Comment:
I think that even for comment text that is short enough to fit on one line, these type of "user documentation" block comments should still be multiline, like so:

```suggestion
  /**
  * If the tooltip should have a small arrow or not. Defaults to 'true'
  */
```

To be clear, this is a matter of personal taste and not a technical requirement, But I think it makes the comment more "balanced" and the closing token will always be in the same place.